codec-http2/src/main/java/io/netty/handler/codec/http2/AbstractHttp2StreamFrame.java

```diff
@@ -22,17 +22,18 @@
  */
 @UnstableApi
 public abstract class AbstractHttp2StreamFrame implements Http2StreamFrame {
-    private Object stream;
```

Comment:
I had it initially as an `Http2Stream` object but changed it to stream id, cause there is little use for `Http2Stream`. I think eventually we will want a `Http2DifferentStream` object where we can attach properties. I ll make that change in a follow up PR.